packages/connect/src/api/bitcoin/refTx.ts

```diff
@@ -78,153 +78,219 @@ const enhanceTransaction = (refTx: RefTransaction, srcTx: BitcoinJsTransaction)
 };
 
 // Transform orig transactions from Blockbook (blockchain-link) to Trezor format
-export const transformOrigTransactions = (
-    txs: TypedRawTransaction[],
-    coinInfo: CoinInfo,
-    addresses?: AccountAddresses,
-): RefTransaction[] =>
-    txs.flatMap(raw => {
-        if (coinInfo.type !== 'bitcoin' || raw.type !== 'blockbook' || !addresses) return [];
-        const { hex, vin, vout } = raw.tx;
-        const tx = BitcoinJsTransaction.fromHex(hex, { network: coinInfo.network });
-        const inputAddresses = addresses.used.concat(addresses.change).concat(addresses.unused);
+const transformOrigTransaction = (
```

Comment:
this is a nitpick, but I like when transforming function define return type. previously there was `RefTransaction[]`
I understand that return type has changed, but is there reason to remove it completely here?

Reply:
now it's returning single `RefTransaction` which is a return type of `enhanceTransaction`

i think it is unnecessary but no problem, i will add it

Reply:
[10a24a5](https://github.com/trezor/trezor-suite/pull/9040/commits/10a24a50842e2cad020670556d6bb503ffd5ebeb)